tests/unit/plugins/lookup/test_revbitspss.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# (c) 2020, Adam Migus <adam@migus.org>
+# (c) 2020, RevBits <info@revbits.com>
```

Comment:
I guess this should be 2021. Also you cannot simply remove the other copyright notice, you have to keep it.

Reply:
Hi @felixfontein ,
                             I am removing this file, and will add our own test cases.